apps/fluent-tester/src/FluentTester/TestComponents/Shadow/ShadowDepthTestSection.tsx

```diff
@@ -1,53 +1,71 @@
 import * as React from 'react';
 import { commonTestStyles } from '../Common/styles';
-import { View } from 'react-native';
+import { ColorValue, View } from 'react-native';
 import { Text } from '@fluentui/react-native';
 import { Shadow, ShadowDepth } from '@fluentui-react-native/experimental-shadow';
-import { Theme, useTheme } from '@fluentui-react-native/theme-types';
+import { useTheme } from '@fluentui-react-native/theme-types';
 import { themedStyleSheet } from '@fluentui-react-native/themed-stylesheet';
+import { useFluentTheme } from '@fluentui-react-native/framework';
 
-const getThemedStyles = themedStyleSheet((t: Theme) => {
+const getThemedStyles = themedStyleSheet(() => {
   return {
     effectBox: {
       width: 366,
       minHeight: 64,
-      backgroundColor: t.colors.background,
     },
     padding: {
       padding: 20,
       paddingHorizontal: 24,
     },
     vmargin: {
-      marginVertical: 32,
+      marginVertical: 16,
+      marginHorizontal: 32,
     },
   };
 });
 
 interface ShadowTestBoxProps {
   depth: ShadowDepth;
+  backgroundColor: ColorValue;
 }
 
 const ShadowTestBox: React.FunctionComponent<ShadowTestBoxProps> = (props: ShadowTestBoxProps) => {
-  const theme = useTheme();
-  const themedStyles = getThemedStyles(theme);
+  const themedStyles = getThemedStyles(useTheme());
   return (
     <Shadow depth={props.depth}>
-      <Text style={[commonTestStyles.view, themedStyles.effectBox, themedStyles.vmargin, themedStyles.padding]} variant="bodySemibold">
-        Shadow Depth: {props.depth}
+      <Text
+        style={[
+          commonTestStyles.view,
+          themedStyles.effectBox,
+          themedStyles.vmargin,
+          themedStyles.padding,
+          { backgroundColor: props.backgroundColor },
+        ]}
+        variant="bodySemibold"
+      >
+        {props.depth}
       </Text>
     </Shadow>
```

Comment:
Would be nice if we can print the key and ambient shadow tokens as well as depth on line 46, but not a blocking requirement. Just a thought :)

Reply:
Just added, will update screenshots